prow/cluster/jobs/istio/test-infra/istio.test-infra.trusted.master.yaml

```diff
@@ -166,7 +166,7 @@ periodics:
       - name: PLANK_DEPLOYMENT_FILE
         value: prow/cluster/plank_deployment.yaml
       - name: COMPONENT_FILE_DIR
-        value: prow/cluster,prow/config/jobs
+        value: prow/cluster,prow/config/jobs,.
```

Comment:
This is intentional?

Reply:
Yes, to include top-level dir which includes `.prow.yaml`